eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/configuration/MongoConfiguration.java

```diff
@@ -56,7 +58,9 @@ public MongoClient mongoClient(MongoProperties properties, ObjectProvider<MongoC
         } else {
             mongoClientOptionsBuilder = new MongoClientOptions.Builder();
         }
-        mongoClientOptions = mongoClientOptionsBuilder.readPreference(ReadPreference.valueOf(readPreference)).build();
+        mongoClientOptions = mongoClientOptionsBuilder.readPreference(ReadPreference.valueOf(readPreference))
```

Comment:
Shall we just hardcode the read preference to `primaryPreferred`? The error message recommends using those settings anyway so...

Reply:
this mongo configuration is reused in the eva-accession-import, the eva-accession-pipeline and the eva-accession-ws. in the pipeline it might make sense as well, but I wouldn't say so for the webservices. I would keep it as it is